openpype/hosts/houdini/plugins/publish/validate_houdini_license_category.py

```diff
@@ -1,32 +1,37 @@
 # -*- coding: utf-8 -*-
 import pyblish.api
 from openpype.pipeline import PublishValidationError
+import hou
 
 
-class ValidateHoudiniCommercialLicense(pyblish.api.InstancePlugin):
-    """Validate the Houdini instance runs a Commercial license.
+class ValidateHoudiniNotApprenticeLicense(pyblish.api.InstancePlugin):
+    """Validate the Houdini instance runs a non Apprentice license.
 
-    When extracting USD files from a non-commercial Houdini license, even with
-    Houdini Indie license, the resulting files will get "scrambled" with
-    a license protection and get a special .usdnc or .usdlc suffix.
+    USD ROPs:
+        When extracting USD files from an apprentice Houdini license,
+        the resulting files will get "scrambled" with a license protection
+        and get a special .usdnc suffix.
 
-    This currently breaks the Subset/representation pipeline so we disallow
-    any publish with those licenses. Only the commercial license is valid.
+        This currently breaks the Subset/representation pipeline so we disallow
+        any publish with apprentice license.
 
+    Alembic ROPs:
+        Houdini Apprentice does not export Alembic.
     """
 
     order = pyblish.api.ValidatorOrder
-    families = ["usd"]
+    families = ["usd", "abc"]
     hosts = ["houdini"]
-    label = "Houdini Commercial License"
+    label = "Houdini Apprentice License"
 
     def process(self, instance):
 
-        import hou
+        if hou.isApprentice():
+            families = [instance.data["family"]]
+            families += instance.data.get("families", [])
+            families = " ".join(families).title()
 
```

Comment:
```suggestion
            # Find which family was matched with the plug-in
            families = {instance.data["family"]}
            families.update(instance.data.get("families", []))
            disallowed_families = families.intersection(self.families)
            families = " ".join(sorted(disallowed_families)).title()
```

Reply:
both solutions work
![image](https://github.com/ynput/OpenPype/assets/20871534/156b1342-5479-4210-86d9-f0f1d1b17f9b)

Reply:
Great.
Make sure to lowercase the "P" on "Publishing" by the way.

Reply:
what about ? 

```suggestion
            # Find which family was matched with the plug-in
            families = {instance.data["family"]}
            families.update(instance.data.get("families", []))
            families = " ".join(families).title()
```
![image](https://github.com/ynput/OpenPype/assets/20871534/9264f44e-9db9-49e6-8268-cf5bafa956be)

Reply:
You still have `Pointcache` in your logs in that last example - which is what we're trying to exclude.

Reply:
I missed that `Pointcache` is mentioned in the error label hehe :D

Reply:
Unresolving this as the issue still stands - and my code snippet shared above should still be a valid fix.

Reply:
Thanks!